examples/resources/genesyscloud_routing_utilization/resource.tf

```diff
@@ -23,4 +23,13 @@ resource "genesyscloud_routing_utilization" "org-utililzation" {
     include_non_acd           = false
     interruptible_media_types = ["call", "chat"]
   }
+  label_utilizations {
```

Comment:
can we have this representation to be array of label_utilization , instead of having multiple attributes with the same name ?

Reply:
ok. I think its fine.